test/unit/component/parser/json/BpmnJsonParser.sequenceFlow.test.ts

```diff
@@ -50,6 +51,7 @@ describe('parse bpmn as json for sequence flow', () => {
       bpmnElementName: 'label 1',
       bpmnElementSourceRefId: 'sourceRef_id_xsdas',
       bpmnElementTargetRefId: 'targetRef_RLk',
+      bpmnElementKind: SequenceFlowKind.NORMAL,
```

Comment:
:question: to limit code changes, could the `ExpectedShape` interface take `SequenceFlowKind.NORMAL` as default value for the `bpmnElementKind` attribute?

Reply:
I tried, but we can't add default value to the interface. But maybe I can add a default value in the method who use it as parameter.
I'll try tomorrow

Reply:
I modified the test method to consider the sequence flow as Normal, when the field is undefined